libevm/VM.cpp

```diff
@@ -815,7 +815,93 @@ void VM::interpretCases()
 			m_sp -= 2;
 		CASE_END
 
+		CASE_BEGIN(JUMPTO)
+		THROW_IF_SUBS_DISABLED();
+			ON_OP();
+			updateIOGas();
+			{
+				uint64_t dest       = m_code[m_pc++];
```

Comment:
For now have moved the extension of the code array to after the jumpdest/beginsub scan, so we know how many extra bytes there are.